src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -1265,10 +1265,10 @@ export default class DataTable extends Vue {
     const ascending: boolean = direction === 'ascending';
 
     if (columnData) {
-      const columnIndex = Object.keys(this.data.head).indexOf(column);
+      const columnIndex = !Array.isArray(this.data.head) ? Object.keys(this.data.head).indexOf(column) : column;
```

Comment:
@dani-cl-madrid We've been experimenting with a new type for the header, being that a DataTableColumn array. This was done so columnCustomization was easier, since the chiToolbarColumnsChange event returns this kind of array. In this case, column will be a number.

Reply:
In that case, you might need to update the definition of `column` so it might be `string | number` instead of just `string`.